src/components/Assay/AssayLineGraph.tsx

```diff
@@ -111,13 +106,90 @@ class AssayLineGraph extends React.Component<AssayLineProps, AssayLineState> {
       elements: { point: { radius: 0 } },
       showLines: true
     };
-    return (
+
+    lockedAssays.forEach((lockedAssay, i) => {
+      const data: any = {
+        datasets: []
+      };
+
+      for (let j = 0; j < activeSubstances.length; j++) {
+        data.datasets = data.datasets.concat(
+          this.createLine(activeSubstances[j], lockedAssay, data.datasets.length));
+      }
+
+      const title = i === 0 ? chartTitle : null;
+
+      const legend = i === lockedAssays.length - 1 ? chartLegend : {
+        display: false
+      };
+
+      let height = lockedAssays.length < 3 ?
```

Comment:
Checking this because `3` assays is the first time there's a graph with neither a title nor a legend?

Reply:
That was it, but it was also just a bunch of playing with it. Since the heights always end up needing to be very slightly different with different numbers of graphs, I probably could have worked out a more complex formula, or done it even more explicitly case-by-case, but blocking into just two cases seemed to work pretty well.